ipatests/test_integration/test_client.py

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (C) 2019  FreeIPA Contributors see COPYING for license
+#
+import random
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+
+
+class TestSSHFromClient(IntegrationTest):
+    """
+    Tests ssh from client when ldap_deref_threshold=0 is added to sssd.conf
+    Automated test for https://pagure.io/SSSD/sssd/issue/3979
+    """
+    num_clients = 1
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master)
+        cls.client = cls.clients[0]
+        tasks.install_client(cls.master, cls.client)
```

Comment:
You don't need to install a client in order to reproduce the issue. You can ssh from the controller host.
This means that you can simplify your test and require only a master installation. In this case, move your code to the existing test ipatests/test_integration/test_commands.py as it's already setting up a 1-master topo, in the class TestIPACommand.

Reply:
But the controller won't be a IPA client right?

Reply:
The controller won't be an IPA client but the bug also happens when ssh'ing from any host with login/passwd. No need to be ssh'ing from a client to reproduce the issue.

Reply:
OK, will make the necessary changes.